core/src/main/scala/spinal/core/Data.scala

```diff
@@ -422,6 +422,15 @@ trait Data extends ContextUser with NameableByComponent with Assignable with Spi
   def assignFromBits(bits: Bits, hi: Int, low: Int): Unit
   def assignFromBits(bits: Bits, offset: Int, bitCount: BitCount): Unit = this.assignFromBits(bits, offset + bitCount.value - 1, offset)
 
+  def clearAll(): this.type = {
```

Comment:
I think it would be great to provide clearAll setAll implementation to the Bool class too, to reduce the number of nodes in the netlist :)

Reply:
Sorry, I somehow missed your comment in my inbox. Pushed the commit addressing Bool.